### PR TITLE
Add proposal

### PR DESCRIPTION
**Pull request by VanhGer**

### CODE REVIEW

File: applications/zk_sealed_bid_auction.md [hunk 1]
```diff
@@ -0,0 +1,262 @@
+
```
Comment (zk-steve) [inline applications/zk_sealed_bid_auction.md]: .

File: applications/zk_sealed_bid_auction.md [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+#  SilentBid - Bringing Sealed-Bid Auction On-chain
+
+
```
Comment (zk-steve) [inline applications/zk_sealed_bid_auction.md:2]: .

File: applications/zk_sealed_bid_auction.md [hunk 3]
```diff
@@ -0,0 +1,262 @@
...
+## [Section 1] Project Information
+
+-  **Project Name:** SilentBid - On-chain sealed-bid highest auction
+
```
Comment (zk-steve) [inline applications/zk_sealed_bid_auction.md]: .

File: applications/zk_sealed_bid_auction.md [hunk 4]
```diff
@@ -0,0 +1,262 @@
...
+-  **Payment Details:** 0x9efE53369D4894Ca22C63f489023B324C3D519BA (on Polygon Network)
+
```
Comment (zk-steve) [inline applications/zk_sealed_bid_auction.md]: .

File: applications/zk_sealed_bid_auction.md [hunk 5]
```diff
@@ -0,0 +1,262 @@
...
+
+
+## [Section 2] Project Overview :page_facing_up:
+
+  
+
+-  **Brief Description:**
+
+This project is a **Sealed-Bid Auction** platform built on **zkVM**, leveraging the **Plonky3** toolkit. Its primary goal is to enable secure and private auctions using **zero-knowledge proofs (ZKPs)** to select the winning bidder without revealing the individual bid amounts, while ensuring fairness in determining the highest bid.
```
Comment (zk-steve) [inline applications/zk_sealed_bid_auction.md]: zkVM ?

File: applications/zk_sealed_bid_auction.md (hunk 1 above)
Comment (VanhGer) [reply applications/zk_sealed_bid_auction.md]: why just a dot ?

File: applications/zk_sealed_bid_auction.md [hunk 6]
```diff
@@ -0,0 +1,262 @@
...
+Auction flow:
+
+![auction_flow](https://raw.githubusercontent.com/VanhGer/soulforge/refs/heads/vjp/auction_flow.png)
+
+Prover & Verifier flow:
+
+![prover_verifier_flow](https://raw.githubusercontent.com/VanhGer/soulforge/refs/heads/vjp/prover_verifier_flow.png)
+
+
+Here is the pseudo code of the proving service:
```
Comment (zk-steve) [inline applications/zk_sealed_bid_auction.md]:
```suggestion
Here is the pseudo code of the circuit:
```

File: applications/zk_sealed_bid_auction.md [hunk 7]
```diff
@@ -0,0 +1,262 @@
...
+
+This architecture ensures that bidders maintain full privacy while the smart contract verifies the proof generated by the proving service. The solution is designed to scale efficiently, ensuring that even with a high number of participants, the privacy of each bid amount is preserved, and the auction remains **transparent** and **fair**. Beyond addressing the practical application of zero-knowledge proofs in sealed-bid auctions, this project serves as a **reference model** for developers seeking to explore **ZK-based decentralized applications using Plonky3**.
+  
+
+-  **Technology Stack:**
+**SilentBid** utilizes a diverse technology stack to build a secure and efficient sealed-bid auction platform:
+
+- Smart Contract: Solidity
+- Proving Service: Rust, Plonky3.
+- Layer 2 Chain: Polygon.
```
Comment (zk-steve) [inline applications/zk_sealed_bid_auction.md]:
```suggestion
      - Smart Contract: Solidity
      - Proving Service: Rust, Plonky3.
      - Layer 2 Chain: Polygon.
```

File: applications/zk_sealed_bid_auction.md [hunk 8]
```diff
@@ -0,0 +1,262 @@
...
+
+- **Proving Service**: Powered by **Plonky3**, this service generates a **zero-knowledge proof** from the **execution trace** of a program that decrypts the encrypted bids and computes the winner, without disclosing the bid amounts.
+
+- **SmartContract**: This contract is responsible for verifying the ZK proof, as well as managing the entire auction lifecycle, including setup, bidding, and settlement.
+
+This architecture ensures that bidders maintain full privacy while the smart contract verifies the proof generated by the proving service. The solution is designed to scale efficiently, ensuring that even with a high number of participants, the privacy of each bid amount is preserved, and the auction remains **transparent** and **fair**. Beyond addressing the practical application of zero-knowledge proofs in sealed-bid auctions, this project serves as a **reference model** for developers seeking to explore **ZK-based decentralized applications using Plonky3**.
+  
+
+-  **Technology Stack:**
+**SilentBid** utilizes a diverse technology stack to build a secure and efficient sealed-bid auction platform:
```
Comment (zk-steve) [inline applications/zk_sealed_bid_auction.md]:
```suggestion

```

File: applications/zk_sealed_bid_auction.md [hunk 9]
```diff
@@ -0,0 +1,262 @@
...
+
+## [Section 2] Project Overview :page_facing_up:
+
+  
+
+-  **Brief Description:**
+
+This project is a **[Sealed-Bid Auction](https://www.investopedia.com/terms/s/sealed-bid-auction.asp)** platform, built leveraging the **Plonky3** toolkit. Its primary goal is to enable secure and private auctions using **zero-knowledge proofs (ZKPs)** to select the winning bidder without revealing individual bid amounts, ensuring both privacy and fairness in determining the highest bid.
+
+What excites us about this project is **its application of zero-knowledge techniques** to address real-world challenges where privacy, transparency, and security are paramount. The platform ensures bid confidentiality, making it suitable for auctions where the bid amounts need to remain private but not necessarily the bidders' identities. This is especially relevant for auctions such as high-value asset sales or certain corporate procurement processes, where protecting the bid amount while maintaining transparency of participants still ensures a fair and secure auction process.
```
Comment (zk-steve) [inline applications/zk_sealed_bid_auction.md]:
```suggestion

```

File: applications/zk_sealed_bid_auction.md [hunk 10]
```diff
@@ -0,0 +1,262 @@
...
+
+Beyond its practical use, this project also serves as a **reference model** for developers interested in building ZK-based decentralized applications using **Plonky3**. It demonstrates how zero-knowledge technology can enhance privacy in bidding scenarios and lays a foundation for future projects in the ZK space.
```
Comment (zk-steve) [inline applications/zk_sealed_bid_auction.md]:
```suggestion
Beyond its practical use, this project also serves as a **reference model** for developers interested in building full-fledged ZK-based decentralized applications using **Plonky3**.
```

File: applications/zk_sealed_bid_auction.md [hunk 11]
```diff
@@ -0,0 +1,262 @@
...
+
+Key components of our project include:
+
+- **Proving Service**: Powered by **Plonky3**, this service generates a **zero-knowledge proof** from the **execution trace** of a program that decrypts the encrypted bids and computes the winner, without disclosing the bid amounts.
+
+- **SmartContract**: This contract is responsible for verifying the ZK proof, as well as managing the entire auction lifecycle, including setup, bidding, and settlement.
+
+This architecture ensures that bidders maintain full privacy while the smart contract verifies the proof generated by the proving service. The solution is designed to scale efficiently, ensuring that even with a high number of participants, the privacy of each bid amount is preserved, and the auction remains **transparent** and **fair**. Beyond addressing the practical application of zero-knowledge proofs in sealed-bid auctions, this project serves as a **reference model** for developers seeking to explore **ZK-based decentralized applications using Plonky3**.
```
Comment (zk-steve) [inline applications/zk_sealed_bid_auction.md]:
```suggestion

```

File: applications/zk_sealed_bid_auction.md [hunk 12]
```diff
@@ -0,0 +1,262 @@
...
+-  **Payment Details:** 0x9efE53369D4894Ca22C63f489023B324C3D519BA (on Polygon Network)
+
+-  **Total Amount Requested:** 10,000 USDC
+
+
+
+## [Section 2] Project Overview :page_facing_up:
+
+  
+
```
Comment (zk-steve) [inline applications/zk_sealed_bid_auction.md]:
```suggestion
```

File: applications/zk_sealed_bid_auction.md (hunk 7 above)
Comment (zk-steve) [inline applications/zk_sealed_bid_auction.md]: - Layer 2 Chain: Polygon.?

**Pull request closed without merging**